BUILDING

```diff
@@ -0,0 +1,60 @@
+Installing
+----------
+
+To build and install tarsnap, run:
+    ./configure
+    make
+    make install
+
+Tarsnap should build and run on any IEEE Std 1003.1 (POSIX) compliant
```

Comment:
huh, lilypond got some workaround for PATH_MAX on HURD a few months ago.  I've made a note to look at this later.